app/models/coronavirus/sub_section.rb

```diff
@@ -13,12 +19,12 @@ def content=(content)
     super.tap { populate_structured_content }
   end
 
-  def featured_link_must_be_in_content
-    return if featured_link.blank? || !structured_content
+  def action_link_present?
+    action_link_url.present? && action_link_content.present? && action_link_summary.present?
+  end
 
-    unless structured_content.links.any? { |l| l.url == featured_link }
-      errors.add(:featured_link, "does not exist in accordion content")
-    end
+  def action_link_blank?
+    action_link_url.blank? && action_link_content.blank? && action_link_summary.blank?
   end
```

Comment:
I don't think we need this method either, we could change the one use of it: https://github.com/alphagov/collections-publisher/pull/1331/files#diff-221c8bf8dbf2b355bc4c0e9cb9ee9e8424bd94bbd3078a19efea804a65d9bba6R70 to just check whether action_link_url is blank